src/BreakpointViewer.h

```diff
@@ -0,0 +1,66 @@
+#ifndef BREAKPOINTVIEWER_H
+#define BREAKPOINTVIEWER_H
+
+#include <QTabWidget>
+#include <QList>
+#include "ui_BreakpointViewer.h"
+#include "DebuggerData.h"
+
+class QPaintEvent;
+class Breakpoints;
+
+class BreakpointViewer : public QTabWidget, private Ui::BreakpointViewer
+{
+	Q_OBJECT;
+public:
+	BreakpointViewer(QWidget* parent = nullptr);
+	void setBreakpoints(Breakpoints* bps);
+
+	enum Type { BREAKPOINT = 0, WATCHPOINT };
```

Comment:
Detail:
In general we either give all of the enum-names an explicit numeric value, or none of them.
It's fine to assume numbering starts at zero and uses consecutive values.